profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module

```diff
@@ -427,6 +430,16 @@ function nexteuropa_multilingual_path_update($path) {
  *     - language: The language of the alias.
  */
 function _nexteuropa_multilingual_path_und_sync($path) {
+
+  // Check if entity has a translation mechanism other than entity translation.
+  $node = menu_get_object("node", 1, $path['source']);
```

Comment:
1) OK, we are dealing with nodes only. Should perhaps be mentioned in the comments.
2) Are we sure $path['source'] is always set?